satpy/tests/writer_tests/test_mitiff.py

```diff
@@ -197,22 +197,22 @@ def test_simple_write(self):
         """Test basic writer operation."""
         from satpy.writers.mitiff import MITIFFWriter
         dataset = self._get_test_dataset()
-        w = MITIFFWriter(mitiff_dir=self.base_dir)
-        w.save_dataset(dataset, writer='mitiff', mitiff_dir=self.base_dir)
+        w = MITIFFWriter()
+        w.save_dataset(dataset, writer='mitiff', base_dir=self.base_dir)
```

Comment:
I don't understand how this is passing the tests. The `save_dataset` call doesn't accept the `base_dir` keyword argument, the `__init__` does. I don't think your tests are actually writing data to `base_dir`.

Reply:
Yes, I agree. I mixed up calling save_dataset from a resampled scene and directly from MITIFFWriter. I will fix.

By the way ... I dont think the tests are actually run?

Reply:
It looks like they are running to me.

Reply:
ok